README.md

```diff
@@ -530,7 +530,7 @@ described in [RFC 8032]
  * [Gitter](https://gitter.im/crystal-lang/crystal)
  * [Google Group](https://groups.google.com/forum/?fromgroups#!forum/crystal-lang)
  * [IRC](http://irc.lc/freenode/crystal-lang) - #crystal-lang on Freenode
- * [Metaruby](https://metaruby.com/c/crystal-forum) - Crystal Forum on Metaruby
+ * [Crystal on Devtalk](https://devtalk.com/crystal) - Crystal Forum on Devtalk.com (was [Metaruby](https://metaruby.com/c/crystal-forum) (see [accouncement](https://metaruby.com/t/goodbye-metaruby-hello-devtalk-com/2284/2))
```

Comment:
Actually with the name of this changing, it needs to be re-alphabetized.

Reply:
Updated, thank you for the feedback! I don't see this listed on the [official Community page](https://crystal-lang.org/community/); should it be demoted to the "Unofficial" section?

Reply:
Thinking more on it, is there a reason to not just replace it with an entry to the actual official forums? I.e. https://forum.crystal-lang.org/.

Then we could move this `devtalk.com` thing down to `Unofficial`, or even just remove it...

Reply:
Actual forums sounds good to me; they are awesome, like this list.

But it's your list, defer to what you guys want.

I'd vote for removing this entirely; it's not very active (although this list might help, but we have the official forums as you say...). I was only updating it b/c I was going through this list and saw it was a broken link.

Reply:
@may I'd be fine with adding official forums and just remove this devtalk thing.

Reply:
@Blacksmoke16 See #599; thank you @veelenga for being so fast on merge, you got to it before I could finish updating per feedback.

It's up to ya'll how you want to handle Devtalk vs. Official now.